react/features/participants-pane/components/native/ParticipantsPaneButton.js

```diff
@@ -34,7 +36,7 @@ class ParticipantsPaneButton extends AbstractButton<Props, *> {
     _handleClick() {
         const { dispatch } = this.props;
 
-        dispatch(open());
+        return ConferenceNavigationContainerRef.navigate('Participants') && dispatch(open());
```

Comment:
There should be no need to dispatch open

Reply:
Removed.